arch/lib/Makefile

```diff
@@ -155,7 +157,7 @@ define cmd_lib_bounds
 	 echo "#define GENERATED_BOUNDS_H"; \
 	 echo ""; \
 	 echo "#define NR_PAGEFLAGS (__NR_PAGEFLAGS)"; \
-	 echo "#define MAX_NR_ZONES (__MAX_NR_ZONES)"; \
+	 echo "#define MAX_NR_ZONES 3"; \
```

Comment:
Thanks for the comment. But how to handle this? I don't quite understand.

On Sun, Sep 6, 2015 at 7:34 PM, Hajime Tazaki <notifications@github.com>
wrote:

> In arch/lib/Makefile
> <https://github.com/libos-nuse/net-next-nuse/pull/47#discussion_r38827642>
> :
>
> > @@ -155,7 +157,7 @@ define cmd_lib_bounds
> >  	 echo "#define GENERATED_BOUNDS_H"; \
> >  	 echo ""; \
> >  	 echo "#define NR_PAGEFLAGS (__NR_PAGEFLAGS)"; \
> > -	 echo "#define MAX_NR_ZONES (__MAX_NR_ZONES)"; \
> > +	 echo "#define MAX_NR_ZONES 3"; \
>
> if possible, exclude this hard-corded value and put it on somewhere
> appropriate like .config.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libos-nuse/net-next-nuse/pull/47/files#r38827642>.
>